src/components/Sum/Sum.jsx

```diff
@@ -1 +1,5 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <>
```

Comment:
You don't need this `<></>` wrapper, because your component has only one `<p><p/>` tag. Maybe that's a reason why  some tests are failed